api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -96,10 +99,28 @@ message DownstreamTlsContext {
 }
 
 // TLS context shared by both client and server TLS contexts.
-// [#next-free-field: 9]
+// [#next-free-field: 11]
 message CommonTlsContext {
   option (udpa.annotations.versioning).previous_message_type = "envoy.api.v2.auth.CommonTlsContext";
 
+  // Config for CA client plugin to get secrets
+  message CAClientPluginConfig {
+    // plugin name
+    string name = 1;
+
+    // opaque name to identify a secret when workload needs more than one secret
```

Comment:
This needs some more explanation. It's not clear from what is present in this PR why this can't live in `typed_config`.

Reply:
I think I'll just remove it from here for now. Will add it later if needed.